lib/views/create-bug.js

```diff
@@ -198,6 +198,8 @@ function submitBug() {
     return;
   }
 
+  form.querySelector('input[type=submit]').setAttribute('disabled', 'disabled');
```

Comment:
This looks a bit daunting. It would be better to break it up like so:
```js
var submitButton = form.querySelector('input[type=submit]');
submitButton.setAttribute('disabled', 'disabled');
```